### PR TITLE
First 3 pages for the mobile developer guide

### PR DESCRIPTION
**Pull request by MonicaG**
This PR contains the changes for the following pages:
* [Introduction](https://f5ff48-developer-portal-dev-f5ff48-dev.apps.silver.devops.gov.bc.ca/docs/default/Component/mobile-developer-guide/)
  * The content of the Introduction page is new. 
* [Getting Started](https://f5ff48-developer-portal-dev-f5ff48-dev.apps.silver.devops.gov.bc.ca/docs/default/Component/mobile-developer-guide/getting_started/)
  * This content is rewritten from the [mobile signing service wiki page](https://github.com/bcgov/mobile-signing-service/wiki).
* [Distribution Methods](https://f5ff48-developer-portal-dev-f5ff48-dev.apps.silver.devops.gov.bc.ca/docs/default/Component/mobile-developer-guide/distribution_methods/)
  * It is rewritten from [the mobile signing service wiki](https://github.com/bcgov/mobile-signing-service/wiki/Apple-Distribution-Methods) and the [deployment section](https://developer.gov.bc.ca/Mobile-Starter-Kit-Overview#deployment) in the Mobile Starter Kit.
  * Note: The Enterprise method was removed from the new documents on purpose. We can discuss offline if needed

### CODE REVIEW

File: docs/distribution_methods.md [hunk 1]
```diff
@@ -1 +1,39 @@
 # Mobile App Distribution Methods
+
+Mobile apps are distributed either through the public app stores or   MDM InTune. Please review the diagram below to determine the distribution method for your app.
+
+![Diagram of the various distribution methods. If the app is for the general public, it is distributed via the Apple App Store and Google Play Stores. If it is an app for Government Employees, it is distributed via MDM inTunes. If it is an app for Employees and Contractors it is distributed as an unlisted app via the Apple App Store](assets/distribution.drawio.svg)
```
Comment (jenreiher-bcgov) [inline docs/distribution_methods.md:5]: brilliant alt text 😻

File: docs/distribution_methods.md [hunk 2]
```diff
@@ -1 +1,39 @@
...
+
+Public apps are released through the Province's accounts on Apple's App Store and Google's Play Store. You must list your app through the Province's accounts. The developer experience team will assist you in setting up your app on these accounts.  Please refer to the [project initiation](initiate.md) page for the information needed to setup an app. Once setup you will have access to [App Store Connect](https://appstoreconnect.apple.com) and [Google Play Console](https://play.google.com/console/about/) to manage your app.
```
Comment (jenreiher-bcgov) [inline docs/distribution_methods.md:7]: Probably need a link out to or description of "how to get in touch with developer experience team" here?

File: docs/distribution_methods.md [hunk 3]
```diff
@@ -1 +1,39 @@
...
+
+Please consult [Apple's app guidelines](https://developer.apple.com/app-store/review/) and [Google's app guidelines](https://support.google.com/googleplay/android-developer/answer/9859455?hl=en&ref_topic=7072031&sjid=10634496881788336983-NA) when preparing your app's release. It is important to follow their guidelines to ensure a smooth app review process.
+
+Employee apps are distributed through [MDM inTune by the OCIO Device Management Team](https://citz.sp.gov.bc.ca/sites/ES/DS/MDAS/Docs/SitePages/Home.aspx). While historically employee apps only supported iOS devices, it is possible for Android support. If you are planning to create an internal Android app, please contact us. You will be the first app and will need extra setup.
+
+Employee and contractor apps also need extra setup in Apple's App Store Connect. The developer experience team will assist in this setup.
```
Comment (jenreiher-bcgov) [inline docs/distribution_methods.md:15]: we might want a link here as well for those folks that dip in and out of parts of docs and don't read them from top to bottom.

File: docs/getting_started.md [hunk 4]
```diff
@@ -1,2 +1,32 @@
 # Getting Started
```
Comment (jenreiher-bcgov) [inline docs/getting_started.md:1]: I wonder if changing the language here to "How to create a new app" would be better here? I could see us also wanting a "getting started developing" section for new devs coming on to an existing project covering some of the stuff we learned from Apple yesterday.

File: docs/index.md [hunk 5]
```diff
@@ -1,2 +1,19 @@
...
+
+The Developer Experience Team will assist you in your mobile app development journey. While we will not code, design or review your app, we will help you:
+
+* Onboard to the app stores
+* Connect with other mobile dev teams
+* Manage signing certificates
+
+Please book an appointment with us at: [Developer.Experience@gov.bc.ca](mailto:Developer.Experience@gov.bc.ca) at the start of your project. Find more information about what to expect at the meeting in our [Getting Started](getting_started.md) guide.
+
+Curious about mobile apps but not sure if you need one? We are happy to [meet and discuss](mailto:Developer.Experience@gov.bc.ca) your scenario.
```
Comment (jenreiher-bcgov) [inline docs/index.md:13]: re: all my comments above about linking to us - I wonder if we need a "how to contact developer experience" markdown file somewhere that we can consistently link to vs spreading that info around. That way if we ever move to a ticketing system vs email it's easier to maintain the change!
don't let this block this pr, but perhaps something for the backlog if we think it's a good idea

File: docs/getting_started.md (hunk 4 above)
Comment (MonicaG) [reply docs/getting_started.md:1]: I'm not sure about "How to create a new app" as I think that implies something more technical? Like creating a new app in Xcode or Android Studio. I dunno though.

I envisioned this page to discuss what is needed at the onboarding meeting. That includes the info needed to get setup on the Apple and Google stores. Plus finding out what the team is building and if they have the resources in place.

Maybe something like:
* Onboard Meeting
* Project Initiation
* Kickoff meeting
* Something else

Thoughts?

File: docs/index.md (hunk 5 above)
Comment (MonicaG) [reply docs/index.md:13]: I've added a contact page. I've updated the links mentioned in previous comments to use this new page.

**Review by CoffeeGurlNia — CHANGES_REQUESTED** [review]
@kaitlyn-rosenburg

File: docs/distribution_methods.md (hunk 3 above)
Comment (MonicaG) [reply docs/distribution_methods.md]: Added a link to the contact page

File: docs/getting_started.md (hunk 4 above)
Comment (MonicaG) [reply docs/getting_started.md]: renamed to "Project initiation"

File: docs/getting_started.md [hunk 6]
```diff
@@ -1,2 +1,32 @@
...
+1. Please provide any capabilities the Apple app will require, such as push notifications, maps etc... Please refer to [Apple's capabilities documentation](https://developer.apple.com/help/account/reference/supported-capabilities-ios/) for the complete list.
+1. Is the app for both Android and iOS devices or only one platform (e.g. only iOS).
+1. Does the app's intellectual property belong to the ministry or the vendor? 
+1. Please provide this information if the ministry owns the app's intellectual property. Will the code be in GitHub, BitBucket Server, or somewhere else? Can the code be public? If the code will live on the [bcgov GitHub organization](https://github.com/bcgov), then please provide the GitHub Ids of the developers. We will add them to the organization.
+1. Who should have access to manage the app on the app stores and what access level should they have? Please refer to the [App Managment](app_management.md) section for more details.
+1. If yes, have you talked to the [Platform Service Team](https://cloud.gov.bc.ca)?
```
[the remark this answers is not in the document]
Comment (MonicaG) [reply docs/getting_started.md:30]: reworded the sentence

**Comment by MonicaG** [comment]
I've incorporated @CoffeeGurlNia feedback. 

@kaitlyn-rosenburg is this okay to merge?

File: docs/distribution_methods.md (hunk 2 above)
Comment (MonicaG) [reply docs/distribution_methods.md]: added link

File: docs/contact.md [hunk 7]
```diff
@@ -0,0 +1,9 @@
+# Contact us
+
+[Contact the Developer Experience Team](mailto:Developer.Experience@gov.bc.ca).
```
Comment (kaitlyn-rosenburg) [inline docs/contact.md:3]:
```suggestion
[Contact the Developer Experience team](mailto:Developer.Experience@gov.bc.ca).
```

File: docs/index.md [hunk 8]
```diff
@@ -1,2 +1,13 @@
-# Mobile App Development
+# Mobile app development
 
+The Province of B.C. provides many types of mobile apps to the public. From wildlife tracking, to digital ID, to employment services and more. These apps are available for download on [Apple's App Store](https://apps.apple.com/us/developer/his-majesty-the-king-in-right-of-the/id487323275) and [Google's Play Store](https://play.google.com/store/apps/developer?id=Province+of+British+Columbia,+Canada&hl=en_CA&gl=US). The Province also creates private apps used by its employees for their job functions.
+
+The Developer Experience Team will assist you in your mobile app development journey. While we won't code, design or review your app, we'll help you:
```
Comment (kaitlyn-rosenburg) [inline docs/index.md:5]:
```suggestion
The Developer Experience team will assist you in your mobile app development journey. While we won't code, design or review your app, we'll help you:
```

File: docs/getting_started.md [hunk 9]
```diff
@@ -1,2 +1,32 @@
-# Getting Started
+# Project initiation
 
+You have an idea for an app. You have approval from your ministry and you're excited to start building. However, before getting started, book an onboarding meeting with the [Developer Experience Team](contact.md). This meeting will help us set up your project in the mobile developer accounts. To make the most of our time together, gather the information outlined below.
```
Comment (kaitlyn-rosenburg) [inline docs/getting_started.md:3]:
```suggestion
You have an idea for an app and you have approval from your ministry. You're excited to start building. 

Before getting started, book an onboarding meeting with the [Developer Experience team](contact.md). This meeting helps us set up your project in the mobile developer accounts. To make the most of our time together, prepare for the meeting and gather the required information.
```

File: docs/getting_started.md [hunk 10]
```diff
@@ -1,2 +1,32 @@
...
+
+## Information to prepare for your onboard meeting
+
+| General information | Response |
+|------|-------|
+| Product Owner name<sup>1</sup> | |
```
Comment (kaitlyn-rosenburg) [inline docs/getting_started.md:9]: Overall I find this a weird way to format the table and corresponding info, using the superscript.

File: docs/getting_started.md [hunk 11]
```diff
@@ -1,2 +1,32 @@
...
+| App application/bundle ID<sup>3</sup> | | 
+| Apple services<sup>4</sup> | |
+| App platform<sup>5</sup>| |
+| Who owns the code<sup>6</sup> | |
+| Where is the code located<sup>7</sup>|
+| App management access<sup>8</sup>||
+| Does the app need a new backend service<sup>9</sup>| |
+
+
+1. Provide contact information for both the product and technical owners. Ideally, both positions should be government employees.
```
Comment (kaitlyn-rosenburg) [inline docs/getting_started.md:22]: bullets don't have periods at the end

Comment (kaitlyn-rosenburg) [reply docs/getting_started.md:22]: applies to all

File: docs/index.md [hunk 12]
```diff
@@ -1,2 +1,13 @@
-# Mobile App Development
+# Mobile app development
 
+The Province of B.C. provides many types of mobile apps to the public. From wildlife tracking, to digital ID, to employment services and more. These apps are available for download on [Apple's App Store](https://apps.apple.com/us/developer/his-majesty-the-king-in-right-of-the/id487323275) and [Google's Play Store](https://play.google.com/store/apps/developer?id=Province+of+British+Columbia,+Canada&hl=en_CA&gl=US). The Province also creates private apps used by its employees for their job functions.
```
Comment (kaitlyn-rosenburg) [inline docs/index.md:3]:
```suggestion
The B.C. government provides many types of mobile apps to the public, like wildlife tracking and employment services. These apps are available for download on [Apple's App Store](https://apps.apple.com/us/developer/his-majesty-the-king-in-right-of-the/id487323275) and [Google's Play Store](https://play.google.com/store/apps/developer?id=Province+of+British+Columbia,+Canada&hl=en_CA&gl=US). We also create private apps used by employees for their job functions.
```

File: docs/index.md (hunk 8 above)
Comment (kaitlyn-rosenburg) [inline docs/index.md:5]:
```suggestion
The Developer Experience team will assist you in your mobile app development journey. While we won't code, design or review your app, we'll help you:
```

**Review by kaitlyn-rosenburg — CHANGES_REQUESTED** [review]

File: docs/distribution_methods.md [hunk 13]
```diff
@@ -1 +1,39 @@
-# Mobile App Distribution Methods
+# Mobile app distribution methods
+
+Mobile apps are distributed either through the public app stores or MDM InTune. Review the diagram below to determine the distribution method for your app.
```
[the remark this answers is not in the document]
Comment (MonicaG) [reply docs/distribution_methods.md:3]:
```suggestion
Mobile apps are distributed through public app stores or MDM InTune. Your distribution method is determined by who will use your app.
```

@kaitlyn-rosenburg - is it ok if I added 'distributed' to the sentence? It seems like it is missing a word?

File: docs/distribution_methods.md [hunk 14]
```diff
@@ -1 +1,39 @@
...
+
+![Diagram of the various distribution methods. If the app is for the general public, it's distributed via the Apple App Store and Google Play Stores. If it's an app for Government employees, it's distributed via MDM inTunes. If it's an app for Employees and Contractors it is distributed as an unlisted app via the Apple App Store.](assets/distribution.drawio.svg)
+
+Public apps are released through the Province's accounts on Apple's App Store and Google's Play Store. You must list your app through the Province's accounts. The [developer experience team](contact.md) will assist you in setting up your app on these accounts. Refer to the [project initiation](getting_started.md) page for the information needed to setup an app. Once setup is complete, you'll have access to [App Store Connect](https://appstoreconnect.apple.com) and [Google Play Console](https://play.google.com/console/about/) to manage your app.
```
[the remark this answers is not in the document]
Comment (MonicaG) [reply docs/distribution_methods.md:7]: The name on the stores is "His Majesty the King in right of the Province of British Columbia", so I was trying to go with something a little less wordy. I know Karl is looking into changing the name, but I'm not sure the status.

File: docs/distribution_methods.md [hunk 15]
```diff
@@ -1 +1,39 @@
...
+
+Both the Apple App Store and Google Play Store have an app review process. When creating your app, it's important to consider how the app reviewer will interact with your app. For instance, if your app requires a login, the app reviewer will need either an account or access to a demo account. If your internal app can't go through the app review process, inform the [developer experience team](contact.md).
+
+Please consult [Apple's app guidelines](https://developer.apple.com/app-store/review/) and [Google's app guidelines](https://support.google.com/googleplay/android-developer/answer/9859455?hl=en&ref_topic=7072031&sjid=10634496881788336983-NA) when preparing your app's release. It's important to follow their guidelines to ensure a smooth app review process.
+
+Employee apps are distributed through [MDM inTune by the OCIO Device Management Team](https://citz.sp.gov.bc.ca/sites/ES/DS/MDAS/Docs/SitePages/Home.aspx). While historically, employee apps only supported iOS devices, they're now also supported by Android. If you're planning to create an internal Android app, [contact us](contact.md). You'll be the first app and will need extra setup.
```
[the remark this answers is not in the document]
Comment (MonicaG) [reply docs/distribution_methods.md:13]:
```suggestion
Employee apps are distributed through [MDM inTune by the OCIO Device Management Team](https://citz.sp.gov.bc.ca/sites/ES/DS/MDAS/Docs/SitePages/Home.aspx). While historically, employee apps only supported iOS devices, they're now also supported by Android. If you're planning to create an internal Android app, [contact us](contact.md). 
```

@kaitlyn-rosenburg - I removed the part about being the first app

File: docs/getting_started.md (hunk 10 above)
Comment (MonicaG) [reply docs/getting_started.md:11]: Reformatted the page

File: docs/getting_started.md (hunk 11 above)
Comment (MonicaG) [reply docs/getting_started.md]: fixed bullet points in new version

**Comment by MonicaG** [comment]
@kaitlyn-rosenburg - I've applied your suggestions and reformatted the project initiation page

**Pull request merged**